examples/dinput8_hook/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+#![feature(once_cell)]
+#![allow(non_snake_case, unused_variables, unreachable_patterns)]
+
+use std::ffi::c_void;
+use std::sync::LazyLock;
+
+use windows::core::{GUID, HRESULT, PCSTR};
+use windows::s;
+use windows::Win32::Foundation::{BOOL, HINSTANCE};
+use windows::Win32::System::LibraryLoader::{GetProcAddress, LoadLibraryA};
+
+static ENTRY_POINT: LazyLock<DirectInput8Create> = LazyLock::new(|| unsafe {
+    let handle = LoadLibraryA(s!("C:\\Windows\\System32\\dinput8.dll\0")).unwrap();
```

Comment:
I appreciate hardcoding that path may work on 99% of users' computers, but I think a more general approach would be better here, even if it is more convoluted. This is a good example of what I mean:

https://github.com/veeenu/eldenring-practice-tool/blob/1216fb55c5b8f415ac74dea815dd2717981993b4/lib/no-logo/src/lib.rs#L54-L65